docs/changelog.md

```diff
@@ -50,6 +50,10 @@ permalink: /changelog/
   * ⚠️ **This is a breaking API change!** for non-component implementations of `HistoryStorage`.
   * `HistoryStorage` got new API: `deleteVisit`.
 
+* **browser-search**
+  * Imported `list.json` and search plugins from Fennec from 2019-03-29.
+  * Added support for `searchDefault` and `searchOrder`.
```

Comment:
nit: Should we mention the "breaking" change from `List<SearchEngine>` to `SearchEngineList` here? This new data class is nice btw. :)

Reply:
Added breaking change for the `SearchProvider` API.